packages/assets/scss/02-molecules/_emergency-alert.scss

```
@@ -3,14 +3,14 @@
 @use "sass:color" as *;
 
 .ma__emergency-alert {
-  background-color: mix(#fff, $c-warning,90%);
+  background-color: mix($c-white, $c-warning,90%);
   font-size: $fonts-small;
   padding-top: 10px;
   padding-bottom: 10px;
   margin-bottom: 0;
 
   & + & {
-    border-top: 1px solid mix(#fff, $c-warning, 50%);
+    border-top: 1px solid mix($c-white, $c-warning, 50%);

```

```suggestion
    border-top: 1px solid $c-highlight-lighter;

```
the output of the mix function is $c-duckling-yellow-lighter or $c-highlight-lighter, lighter colors are usually used as border colors

---

Done.